qiskit/algorithms/amplitude_amplifiers/grover.py

```diff
@@ -325,3 +334,63 @@ def iterations(self, value: List[int]) -> None:
             value: A new value for the powers.
         """
         self._iterations = value
+
+    @property
+    def circuit_results(self) -> Optional[Union[np.ndarray, Dict[str, int]]]:
+        """Return the circuit results. Can be a statevector or counts dictionary."""
+        return self._circuit_results
+
+    @circuit_results.setter
+    def circuit_results(self, value: Union[np.ndarray, Dict[str, int]]) -> None:
+        """Set the circuit results."""
+        self._circuit_results = value
+
+    @property
+    def top_measurement(self) -> str:
```

Comment:
Can you fix the docstrings of this getter/setter and all that follow since the text that of `circuit_results`.